client/src/HowItWorks.tsx

```diff
@@ -1,7 +1,341 @@
+    title: "We're helping you make a difference in your community.",
+    list: [
+        loremIpsum.slice(0,97),
+        loremIpsum.slice(0,97),
+        loremIpsum.slice(0,97),
+        loremIpsum.slice(0,97),
+    ],
+};
+
+const nonprofitQuestions = [
+    {question: 'Can we post for volunteer help?', answer: loremIpsum.slice(0,150)},
+    {question: 'Can we keep working with a person we exchanged with?', answer: loremIpsum.slice(0,150)},
+    {question: 'How do we see what has been donated previously?', answer: loremIpsum.slice(0,150)},
+];
+const citizenQuestions = [
+    {question: 'Can I offer to volunteer?', answer: loremIpsum.slice(0,150)},
+    {question: 'Can I keep working with a nonprofit I exchanged with?', answer: loremIpsum.slice(0,150)},
+];
+
+const useStyles = makeStyles((theme: Theme) => ({
+    // universal styles
+    header: {
+        fontWeight: 'bold',
+        marginBottom: 15
+    },
+    headerText: {
+        width: '100%',
+        maxWidth: '1100px',
+        fontSize: '1.3rem',
+        textAlign: 'left',
+    },
+    mainPageSection: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'center',
+        paddingTop: '50px',
+        paddingBottom: '50px',
+        paddingLeft: '20px',
+        paddingRight: '20px',
+        fontSize: '1.4rem',
+        '& h3': {
+            fontSize: '1.6em',
+            width: '100%',
+        },
+    },
+    limitWidth: {
+        margin: 'auto',
+        width: '100%',
+        maxWidth: '1100px',
+    },
+    // page specific styles
+    questionSection: {
+        margin: '0 auto',
+        padding: '0 10px',
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'center',
+    },
+    greySection: {
+        backgroundColor: '#C4C4C4',
+    },
+    columns: {
+        display: 'flex',
+    },
+    titleBox: {
+        height: 200,
+        backgroundColor: '#C4C4C4',
+    },
+    tabs: {
+        margin: 'auto',
+        padding: '0',
+        borderBottom: '1px solid #C4C4C4',
+        '& .MuiGrid-item': {
+            padding: '20px',
+            paddingBottom: '0!important',
+        },
+    },
+    grid: {
+    },
+    button: {
+        fontSize: '2rem',
+        margin: '20px 0 0 0',
+        borderBottom: '11px solid #FFFFFF',
+    },
+    hidden: {
+        display: 'none',
+    },
+    selected: {
+        borderBottom: '11px solid #C4C4C4',
+    },
+    nonprofitContainer: {
+        // height: 200,
+        // backgroundColor: 'blue',
+    },
+    citizenContainer: {
+        // height: 200,
+        // backgroundColor: 'red',
+    },
+    questions: {
+        alignSelf: 'flex-end',
+        width: '100%',
+        maxWidth: '700px',
+    },
+}));
 
 function HowItWorks() {
-    return <div>HowItWorks</div>;
+    const classes = useStyles();
+
+    const [ tabSelected, setTabSelected ] = React.useState("nonprofit");
```

Comment:
like this, typescript will set this state as an `any` type
we can take advantage of enums here to enforce that it's only 1 of the 2 possibilities
and then typescript will catch any errors therefore giving us more confidence
so `React.useState<'nonprofit' | 'citizen'>("nonprofit");`

Reply:
Oh that's great!